src/Lean/Elab/Tactic/Simp.lean

```diff
@@ -201,7 +202,7 @@ def elabSimpArgs (stx : Syntax) (ctx : Simp.Context) (simprocs : Simp.SimprocsAr
               arg[0][0].getKind == ``Parser.Tactic.simpPost
           let inv  := !arg[1].isNone
           let term := arg[2]
-          match (← resolveSimpIdTheorem? term) with
+          match ← resolveSimpIdTheorem? term with
```

Comment:
Could you please keep the parentheses here? This is the style we use in the core repo. We write nested actions `<- f a` as `(<- f a)`.

Reply:
Ah, then I have to revise my code elsewhere. I have been omitting it since the syntax doesn't require it in `if` and `match` and `unless`, and I tend to not use unnecessary parentheses.

```suggestion
          match (← resolveSimpIdTheorem?) term with
```